jobs/resource_job.go

```diff
@@ -153,6 +153,10 @@ type JobTaskSettings struct {
 	Description string           `json:"description,omitempty"`
 	DependsOn   []TaskDependency `json:"depends_on,omitempty"`
 
+	// BEGIN Jobs + RunIf preview
```

Comment:
will it be used for some automation later?

Reply:
I saw it on other parts, not sure what it's for.

Reply:
it's not strictly required...